temporal-fixtures/openNclosed/README.md

```diff
@@ -0,0 +1,8 @@
+This fixuture creates 20 workflows - 10 open, 10 closed:
```

Comment:
it's currently going to create either 20 open (for 10 minutes) or 20 immediately closed, depending on the `keep` flag

Reply:
```suggestion
This fixuture creates 20 workflows - all either open for 10 minutes, or all either closed right away, depending on the `keep` flag:
```